sshguard/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+- name: install sshguard
+  apt:
+    pkg: sshguard
+    state: present
+    update_cache: yes
+  tags: ['sshguard', 'sshguard:install']
+
+- name: update sshguard whitelist
+  lineinfile:
+    dest: /etc/sshguard/whitelist
+    regexp: "^{{item}}"
```

Comment:
`{{item}}` -> `{{ item }}`, with spaces just like most of our playbooks' convention.

Reply:
Fixed. Thanks!